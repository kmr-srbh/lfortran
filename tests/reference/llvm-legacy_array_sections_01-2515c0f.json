{
    "basename": "llvm-legacy_array_sections_01-2515c0f",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/legacy_array_sections_01.f90",
    "infile_hash": "90cec79afc88ad4e171e9ff3c6f8612e820b4906e444054346abdf34",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "llvm-legacy_array_sections_01-2515c0f.stderr",
    "stderr_hash": "114ca504160d91e61127d84188eb67a579b792df005b565159aa62d4",
    "returncode": 1
}