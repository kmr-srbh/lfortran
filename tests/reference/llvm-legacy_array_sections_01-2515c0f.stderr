code generation error: Type mismatch in subroutine call, expected `f64[:]`, passed `f64`. Hint: Use the `--legacy-array-sections` flag to compile.
 --> tests/../integration_tests/legacy_array_sections_01.f90:5:8
  |
5 | call b(w(icon))
  |        ^^^^^^^ 
